thrift/agent.thrift

```diff
@@ -23,6 +23,7 @@
 include "jaeger.thrift"
 include "zipkincore.thrift"
 
+namespace cpp uber.jaeger.agent.thrift
```

Comment:
maybe `s/uber.jaeger/jaegertracing/`?

Reply:
Ya probably same applies to https://github.com/isaachier/cpp-client